test/models/thesis_test.rb

```diff
@@ -752,4 +752,17 @@ class ThesisTest < ActiveSupport::TestCase
     change = t.versions.last
     assert_equal change.changeset['title'], %w[MyString updated]
   end
+
+  test 'evaluates whether a thesis is newly created' do
+    preexisting_thesis = theses(:one)
+    assert_equal false, preexisting_thesis.new_thesis?
```

Comment:
You may want to consider using syntax like  `refute(preexisting_thesis.new_thesis?)` for true / false checks in mini_test.

https://minitest.rubystyle.guide/#refute-false

and 

https://minitest.rubystyle.guide/#assert-truthy